docs/content/en/docs/installation/runtime-hooks.md

```diff
@@ -89,27 +89,106 @@ Install the new configuration file and restart containerd
 minikube cp /tmp/new-config.toml /etc/containerd/config.toml
 minikube ssh sudo systemctl restart containerd
 ```
+{{% /tab %}}
+
+{{% tab "kind with Containerd (default)" %}}
+
+```shell
+cat <<EOF > kind-config.yaml
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+  - role: control-plane
+    extraMounts:
+      - hostPath: /proc
+        containerPath: /procHost
+EOF
+kind create cluster --config kind-config.yaml
+EXTRA_HELM_FLAGS=(--set tetragon.hostProcPath=/procHost) # flags for helm install
+```
+
+Tetragon Runtime Hooks use [NRI](https://github.com/containerd/nri). NRI is [enabled by
+default](https://github.com/containerd/containerd/blob/main/docs/NRI.md#disabling-nri-support-in-containerd)
+starting from containerd version 2.0. For version 1.7, however, it needs to be enabled in the
+configuration.
+
+You can check the containerd version using:
+
+```shell
+docker exec -it kind-control-plane crictl version
+```
+
+Output should be similar to:
+
+```text
```

Comment:
you can drop `text` so that it looks like above in this example, it's more clear that it's an output.

<img width="760" alt="image" src="https://github.com/user-attachments/assets/25b1f536-f3ac-4b9b-867d-9b88a935ec96" />